src/components/PropTable.js

```diff
@@ -82,6 +86,7 @@ export default class PropTable extends React.Component {
               <td>{row.propType}</td>
```

Comment:
@phthhieu Since this is pretty related, I like to do that with this PR. May be we need to change the title.